Mathlib/Algebra/Order/GroupWithZero/Unbundled.lean

```diff
@@ -1189,6 +1202,50 @@ lemma div_le_one₀ (hb : 0 < b) : a / b ≤ 1 ↔ a ≤ b := by rw [div_le_iff
 @[deprecated (since := "2024-08-21")] alias div_le_iff := div_le_iff₀
 
 end MulPosMono
+
+section PosMulStrictMono
+variable [PosMulStrictMono G₀]
+
+/-- See `lt_inv_mul_iff₀'` for a version with multiplication on the other side. -/
+lemma lt_inv_mul_iff₀ (hc : 0 < c) : a < c⁻¹ * b ↔ c * a < b where
+  mp h := by simpa [hc.ne'] using mul_lt_mul_of_pos_left h hc
+  mpr h := by simpa [hc.ne'] using mul_lt_mul_of_pos_left h (inv_pos.2 hc)
+
+/-- See `inv_mul_lt_iff₀'` for a version with multiplication on the other side. -/
+lemma inv_mul_lt_iff₀ (hc : 0 < c) : c⁻¹ * b < a ↔ b < c * a where
+  mp h := by simpa [hc.ne'] using mul_lt_mul_of_pos_left h hc
+  mpr h := by simpa [hc.ne'] using mul_lt_mul_of_pos_left h (inv_pos.2 hc)
+
+lemma one_lt_inv_mul₀ (ha : 0 < a) : 1 < a⁻¹ * b ↔ a < b := by rw [lt_inv_mul_iff₀ ha, mul_one]
+lemma inv_mul_lt_one₀ (ha : 0 < a) : a⁻¹ * b < 1 ↔ b < a := by rw [inv_mul_lt_iff₀ ha, mul_one]
+
+lemma one_lt_inv₀ (ha : 0 < a) : 1 ≤ a⁻¹ ↔ a ≤ 1 := by simpa using one_le_inv_mul₀ ha (b := 1)
```

Comment:
This is an `le` lemma called `lt`. I guess you intended to add the `le` version and move the `lt` version but something got mixed up?

Reply:
Got mixed up indeed!